### PR TITLE
Improve 'configuring' docs

### PR DESCRIPTION
**Pull request by adamchainz**
Tidy wording and improve syntax highlighting - single backticks had been used in some places instead of double, giving italics rather than code highlighting.

### CODE REVIEW

File: docs/configuring_django.rst [hunk 1]
```diff
@@ -41,20 +41,23 @@ Example contents of pytest.ini::
...
-configuration is used at the same time, pytest-django will prefer using
-settings from the command line option `--ds`, then the environment variable and
-last the pytest.ini.
-You can use `addopts = --ds=yourtestsettings` in your pytest configuration
-to automatically add the `--ds` option.
+The order of precedence is, from highest to lowest:
+
+* The command line option ``--ds``
+* The environment variable ``DJANGO_SETTINGS_MODULE``
+* The configuration file - ``pytest.ini``, or file that Pytest finds such as
```
Comment (blueyed) [inline docs/configuring_django.rst]: s/Pytest/pytest/

File: docs/configuring_django.rst [hunk 2]
```diff
@@ -41,20 +41,23 @@ Example contents of pytest.ini::
...
+  ``tox.ini``
+
+If you want to override from the configuration file, you can use
+``addopts = --ds=yourtestsettings``.
```
Comment (blueyed) [inline docs/configuring_django.rst]: Mention that `DJANGO_SETTINGS_MODULE` is the name to be used in the config? (when not overriding)

File: docs/configuring_django.rst (hunk 1 above)
Comment (adamchainz) [reply docs/configuring_django.rst]: Done

File: docs/configuring_django.rst (hunk 2 above)
Comment (adamchainz) [reply docs/configuring_django.rst]: Reworded a bit.

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**